web/yo/app/scripts/directives/curationQueue.js

```diff
@@ -212,46 +212,32 @@ angular.module('oncokbApp')
                         addedAt: new Date().getTime(),
                         dueDay: $scope.input.dueDay ? new Date($scope.input.dueDay).getTime() : '',
                         comment: $scope.input.comment,
-                        notified: false
-                    });
+                        notified: false,
+                        hugoSymbol: hugoSymbol
+                    };
                     if ($scope.location === 'gene') {
-                        item.set('variant', $scope.input.variant);
+                        item['variant'] = $scope.input.variant;
                     } else if ($scope.location === 'queues') {
                         if ($scope.data.hugoVariantMapping[hugoSymbol]) {
-                            item.set('variant', $scope.data.hugoVariantMapping[hugoSymbol]);
+                            item['variant'] = $scope.data.hugoVariantMapping[hugoSymbol];
                         } else {
-                            item.set('variant', '');
+                            item['variant'] = '';
                         }
                     }
                     if ($scope.predictedArticle && $scope.validPMID) {
-                        item.set('article', $scope.predictedArticle);
-                        item.set('pmid', $scope.input.article);
+                        item['article'] = $scope.predictedArticle;
+                        item['pmid'] = $scope.input.article;
+                        item['pmidString'] = 'PMID: ' + item['pmid'];
                     } else {
-                        item.set('article', $scope.input.article);
-                    }
-                    $rootScope.queuesData.get(hugoSymbol).push(item);
-                    var queueItem = {
-                        article: item.get('article'),
-                        pmid: item.get('pmid'),
-                        pmidString: 'PMID: ' + item.get('pmid'),
-                        link: item.get('link'),
-                        variant: item.get('variant'),
-                        mainType: item.get('mainType'),
-                        subType: item.get('subType'),
```

Comment:
why are those logics removed?

Reply:
Previously we need two variable item and queueItem, one is JSON object, the other is the Map object for the queue model in Google Drive. Now we don't need the second one any more so the logic is removed.